src/main/java/redis/clients/jedis/json/parser/JsonParser.java

```diff
@@ -0,0 +1,26 @@
+package redis.clients.jedis.json.parser;
+
+/**
+ * Represents the ability of serialize an object to JSON format string
+ * and deserialize it to the typed object.
+ *
+ * @see DefaultGsonParser Default implementation for JSON parser engine with com.google.gson.Gson
+ */
+public interface JsonParser {
```

Comment:
This represents both a serializer and deserializer. The word parser generally refers to deserializer. Is it possible to have another name?

@chayim @uglide Any suggestion?

Reply:
Yes. You are right.
I've some suggestions.

1. RedisJsonMapper;
2. JsonMapper;
3. RedisObjectMapper;
4. ObjectMapper;
5. RedisParseGenerateJson;
6. RedisConvertObjectJson;
7. RedisJson;
8. Redison;
9. Rson;

Reply:
1-4 are reasonable. One more to consider: `JsonObjectMapper`.